### PR TITLE
Add SSL support with certificate checking.

### PR DESCRIPTION
**Pull request by aguynamedben**

### CODE REVIEW

File: simplegeo/https.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+        def getresponse(self):
+            if not self._exc_info:
+                return httplib2.HTTPSConnectionWithTimeout.getresponse(self)
+            else:
+                raise self._exc_info[1], None, self._exc_info[2]
+    return ValidatingHTTPSConnection
+
+
+def request(url,
```
Comment (aguynamedben) [inline simplegeo/https.py:61]: I think I need to pass the certificates to this function to make it actually check them.  keyfile, certfile, or ca_certs.  Not sure though.

**Comment by mihasya** [comment]
Weak.

**Comment by aguynamedben** [comment]
WILL NOT MERGE

**Comment by aguynamedben** [comment]
:sparkles:

**Pull request closed without merging**